djangoproject/settings.py

```diff
@@ -151,6 +151,7 @@
 AUTH_USER_MODEL = "IMID.CustomUser"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = os.path.join(BASE_DIR, "uploaded")
+# MEDIA_ROOT = os.path.join(BASE_DIR, "uploaded")
+MEDIA_ROOT = os.path.join("/data/mingcanIMID/", "uploaded")
```

Comment:
This needs to be reversed - it's a value of MEDIA_ROOT that's specific to your local env but not relevant for prod.

Reply:
See my comment above. That isn't the way pull requests work.

Reply:
I think you're getting confused about the difference of having changes in your local dev env and actually committing those changes to the remote repository. The problem is that you keep doing "git add ." (a big no-no), which is adding ALL your changes, even the ones you shouldn't be committing...

Reply:
In the interests of time I've now resolved these myself.